R/warnings.R

```diff
@@ -80,9 +80,7 @@ warn_if_invalid_dtc <- function(dtc, is_valid = is_valid_dtc(dtc)) {
       "2003---15\n\n",
       "The following ISO representations, and any other representation are NOT handled: \n",
       "2003-12-15T-:15:18\n",
-      "2003-12-15T13:-:19\n",
-      "--12-15\n",
-      "-----T07:15"
+      "2003-12-15T13:-:19\n"
```

Comment:
I think we can remove this part completely because now we are handling them.

Maybe we should update the first part of the message to something like "ISO representations of the form YYYY-MM-DDThh:mm:ss.ddd are expected, e.g., 2003-12-15T13:15:17.123. Missing parts at the end can be omitted. Missing parts in the middle must be represented by a dash, e.g., 2003---15.".